test/verify/check-metrics

```diff
@@ -168,6 +168,34 @@ class TestMetrics(MachineCase):
 
         m.execute("kill %d" % net_pid)
 
+        if m.image in ["rhel-8-2-distropkg"]:
+            return  # HACK: individual graph pages broken, fixed in PR #13438
+
+        # CPU graph page
+        b.click("#link-cpu")
+        b.wait_not_visible(".ct-graph-memory")
+        b.wait_visible("#cpu_status_title")
+        b.wait_visible("#cpu_status_graph")
```

Comment:
I thought that we are also going to check that the height (and width) is some reasonable number. Something like:
```
width = b.call_js_func('(function (sel) { return ph_find(sel).offsetWidth; })', "#cpu_status_graph")
```

Reply:
Cheers! I pushed the size check.